cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -381,19 +381,15 @@ fn send_weth_asset_from_asset_hub_to_ethereum() {
 	let assethub_sovereign = BridgeHubRococo::sovereign_account_id_of(assethub_location);
 	let bridgehub_location = AssetHubRococo::sibling_location_of(BridgeHubRococo::para_id());
 
-	AssetHubRococo::force_default_xcm_version(Some(XCM_VERSION));
-	BridgeHubRococo::force_default_xcm_version(Some(XCM_VERSION));
 	AssetHubRococo::force_xcm_version(
 		Location::new(2, [GlobalConsensus(Ethereum { chain_id: CHAIN_ID })]),
 		XCM_VERSION,
 	);
 
-	BridgeHubRococo::fund_accounts(vec![(assethub_sovereign.clone(), INITIAL_FUND)]);
-
```

Comment:
Should remove this initial fund as there is no need to fund the sovereign here any more.